tensorflow_datasets/core/dataset_info.py

```diff
@@ -265,6 +268,41 @@ def _dataset_info_path(self, dataset_info_dir):
   def _license_path(self, dataset_info_dir):
     return os.path.join(dataset_info_dir, LICENSE_FILENAME)
 
+  def visualize_dataset(self):
+    """Visualize dataset statistics with FACETS DIVE.
+
+       Example Usage:
+
+          titanic_builder = tfds.builder("titanic")
+          titatanic_builder.download_and_prepare()
+          titanic_builder.info.visualize_dataset()
+
+       Return a <IPython.core.display.HTML object>
+
+       So it's just use on the jupyter notebook."""
+
+    filetype_suffix = self._builder._file_format_adapter.filetype_suffix
+
+    if filetype_suffix != "tfrecord":
+      raise ValueError(
+        "Cannot visualize dataset statistics because filetype {}".format(
+          filetype_suffix))
+
+    # Find the train .tfrecord files pattern of dataset.
+    filepattern = naming.filepattern_for_dataset_split(
+      self._builder.name, 'train', self._builder.data_dir, filetype_suffix)
+
+    statistics = tfds.core.lazy_imports.tfdv.generate_statistics_from_tfrecord(
```

Comment:
I think there is confusion, in our design, we want to generate the TFDV statistics during dataset creation time, similarly to: https://github.com/tensorflow/datasets/pull/383

Then, the visualisation can be done simply `tfdv.visualize_statistics(info.statistics)`

The main advantage is that as some datasets are huge, we don't want to re-compute statistics every times (ex: iterating over imagenet may takes multiple hours).
Because we are pre-computing and updating the statistics info files on GCS, user can directly download the statistics without having to generate the dataset first. Like:
```
builder = tfds.builder('imagenet_2012')  # Download pre-computed statistics from GCS
builder.info.visualize_dataset()
```

Is your plan to restore  https://github.com/tensorflow/datasets/pull/383 in a new PR ?

Reply:
Yep I see the problem.
Do we pull infos automatically or do users have to download? If it's auto where is it?

Reply:
Also if we keep the statistics on gcs, why compute the statistics again on users computer?

Reply:
> Is your plan to restore #383 in a new PR ?
If I restore it, will we use it ? Because of tfdv problems.